build.gradle

```diff
@@ -1,18 +1,19 @@
 buildscript {
 
     ext.modules = [
-            "sdkVersionName": "0.0.0-test-sdk",
-            "sdkVersionCode": 1,
+            "sdkVersionName"      : "0.0.0-test-sdk",
+            "sdkVersionCode"      : 1,
```

Comment:
Did this pass Lint? Or was it fixed by it?

Reply:
We don't have any linters that inspect our groovy build.gradle files. This formatting style is the default one on Android Studio. I'm open to updating this default if the team doesn't like the look of this!